roles/ceph-osd/tasks/check_mandatory_vars.yml

```diff
@@ -30,6 +30,7 @@
     - not raw_multi_journal
     - not dmcrypt_journal_collocation
     - not dmcrypt_dedicated_journal
+    - not lvm_osds
 
 - name: verify only one osd scenario was chosen
```

Comment:
While I understand the intent behind this task I think in practice it's gonna be very hard to maintain. This check already does not include all of the existing osd scenario combinations. YAML/ansible is just not a great at providing this kind of input validation, is there a better way we can do this?